samples/EmbeddedViewSample.Web/Controllers/EmbeddedView_AdminController.cs

```diff
@@ -3,10 +3,10 @@
 
 using Microsoft.AspNet.Mvc;
 
-namespace RazorEmbeddedViewsWebSite.Controllers
+namespace EmbeddedViewSample.Web.Controllers
 {
     [Area("Restricted")]
-    public class RazorEmbeddedViews_AdminController : Controller
+    public class EmbeddedView_AdminController : Controller
```

Comment:
Can we get rid of this goofy naming as part of this transition? This just makes it a worse sample because it's bad and distracting